From af55fba463ca6fa1e6ab0ebfb60cc083e19f7beb Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 26 Aug 2014 15:30:29 -0400
Subject: [ethernet] igbvf: Convert uses of __constant_<foo> to <foo>

Message-id: <1409067034-17415-9-git-send-email-sassmann@redhat.com>
Patchwork-id: 89219
O-Subject: [RHEL7.1 PATCH 08/13] igbvf: Convert uses of __constant_<foo> to <foo>
Bugzilla: 1091122
RH-Acked-by: Neil Horman <nhorman@redhat.com>
RH-Acked-by: John Greene <jogreene@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: John Linville <linville@redhat.com>

From: Joe Perches <joe@perches.com>

The use of __constant_<foo> has been unnecessary for quite awhile now.

Make these uses consistent with the rest of the kernel.

Signed-off-by: Joe Perches <joe@perches.com>
Tested-by: Aaron Brown <aaron.f.brown@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit 1dcf875effe050013602ebb61a108ce5f19e0459)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/igbvf/netdev.c b/drivers/net/ethernet/intel/igbvf/netdev.c
index e2c6d80..b7ab03a 100644
--- a/drivers/net/ethernet/intel/igbvf/netdev.c
+++ b/drivers/net/ethernet/intel/igbvf/netdev.c
@@ -2014,12 +2014,12 @@ static inline bool igbvf_tx_csum(struct igbvf_adapter *adapter,
 
   if (skb->ip_summed == CHECKSUM_PARTIAL) {
    switch (skb->protocol) {
-   case __constant_htons(ETH_P_IP):
+   case htons(ETH_P_IP):
     tu_cmd |= E1000_ADVTXD_TUCMD_IPV4;
     if (ip_hdr(skb)->protocol == IPPROTO_TCP)
      tu_cmd |= E1000_ADVTXD_TUCMD_L4T_TCP;
     break;
-   case __constant_htons(ETH_P_IPV6):
+   case htons(ETH_P_IPV6):
     if (ipv6_hdr(skb)->nexthdr == IPPROTO_TCP)
      tu_cmd |= E1000_ADVTXD_TUCMD_L4T_TCP;
     break;
-- 
1.7.1